src/test/datascience/kernel-launcher/localKernelFinder.unit.test.ts

```diff
@@ -559,6 +569,31 @@ import { IExtensions } from '../../../client/common/types';
                     });
                     assert.equal(kernel?.interpreter, activeInterpreter);
                 });
+                test('Return conda interpreter if we have conda env name as kernelspec name in notebook metadata', async () => {
```

Comment:
@rchiodo @IanMatthewHuff Added a new test & fixed this.
Not sure how we missed this, but the tests prove that the current search doesn't work properly (basically the preferred interpreter is not as expected).

Reply:
@DonJayamanne Sorry where is the fix in code? I'm not seeing it off hand.